packages/aws-cdk-lib/region-info/lib/aws-entities.ts

```diff
@@ -69,6 +69,7 @@ export const AWS_REGIONS_AND_RULES: readonly (string | symbol)[] = [
   'eu-isoe-west-1', // EU ISO-E West
   'us-isob-west-1', // US ISOB West
   'ap-east-2', // Asia Pacific (Taipei)
+  'eusc-de-east-1', // EU (Germany)
```

Comment:
Let's add an entry for aws-eusc in the partition map below so it matches [this](https://github.com/aws/aws-sdk-js-v3/blob/main/packages/util-endpoints/src/lib/aws/partitions.json#L244C13-L244C21)